lms/templates/bookmarks/bookmarks_results.underscore

```diff
@@ -0,0 +1,42 @@
+<div id="my-bookmarks" class="sr-is-focusable" tabindex="-1"></div>
+<h2 class="bookmarks-results-header" tabindex="0"><%= gettext("My Bookmarks") %></h2>
```

Comment:
@cptvitamin Actually from the second last line of https://github.com/edx/edx-platform/pull/7896#discussion_r29672227 it looke like `My Bookmarks` will also be focusable. Anyway i have removed `tabindex=0` from `<h2>`